packages/ec-push-cli/src/main.rs

```diff
@@ -65,12 +65,16 @@ fn main() -> Result<()> {
                 let state = PushState::builder()
                     .with_max_stack_size(1000)
                     .with_program(program.get_instructions())
-                    // This will return an error if the program is longer than the allowed max stack size.
+                    // This will return an error if the program is longer than
+                    //  the allowed max stack size.
                     // We arguably should check that and return an error here.
                     .unwrap()
                     .with_int_input("x", input)
                     .build();
-                // This is the degree 3 problem in https://github.com/lspector/Clojush/blob/master/src/clojush/problems/demos/simple_regression.clj
+                // This is the degree 3 problem in
+                // https://github.com/lspector/Clojush/blob/
```

Comment:
This (and the following reformatting) is problematic because it breaks up these URLs, making it impossible to copy/paste them out of the code to see what's being referred to.